tests/transforms/test_batch_transform.py

```diff
@@ -238,9 +240,13 @@ def my_transform(self, tape, **kwargs):
         spy_expand.assert_called()  # The expand_fn of transform_fn always is called
 
         input_tape = spy_transform.call_args[0][1]
-        assert len(input_tape.operations) == 1
-        assert input_tape.operations[0].name == ("RZ" if perform_expansion else "PhaseShift")
+
+        assert len(input_tape.operations) == 2 if perform_expansion else 1
+        assert input_tape.operations[0].name == "RZ" if perform_expansion else "PhaseShift"
         assert input_tape.operations[0].parameters == [0.5]
+        if perform_expansion:
+            assert input_tape.operations[1].name == "GlobalPhase"
+            assert input_tape.operations[1].parameters == [0.25]
```

Comment:
Can put it all under one if condition

```suggestion
        if perform_expansion:
            assert len(input_tape.operations) == 2
            assert input_tape.operations[0].name == "RZ"
            assert input_tape.operations[0].parameters == [0.5]
            assert input_tape.operations[1].name == "GlobalPhase"
            assert input_tape.operations[1].parameters == [0.25]
        else: 
            assert len(input_tape.operations) == 1
            assert input_tape.operations[0].name == "PhaseShift"
            assert input_tape.operations[0].parameters == [0.5]
```